### PR TITLE
uiplib/scheduler.py: Feature added to skip wallpapers

### LINKED ISSUE #89: Shortcut to change wallpaper without waiting for timeout
I want a shortcut to change the wallpaper before timeout.
I might not like a currently set random wallpaper. I would like to have an option to change it rather than wait for the timespan.

### PR DESCRIPTION
**Pull request by DarkSouL11**
Now users can skip the current wallpaper by just pressing enter.

Fixes #89

### CODE REVIEW

**Comment by DarkSouL11** [comment]
@nkprince007 please confirm if this is working on mac and @abhsag24 check if its working on windows

File: uiplib/scheduler.py [hunk 1]
```diff
@@ -44,16 +47,39 @@ def change_random(self):
         print("changing desktop wallpaper to: ", path)
         change_background(path)
 
+    def kbhit(self):
+        ''' Returns True if keyboard character was hit, False otherwise.
+        '''
+        if os.name == 'nt':
+            import msvcrt
```
Comment (abhsag24) [inline uiplib/scheduler.py]: imports on top is this a standard library? if not add it to `requirements.txt`

Comment (DarkSouL11) [inline uiplib/scheduler.py]: @abhsag24 moved import to the top.Anything else?

File: uiplib/scheduler.py [hunk 2]
```diff
@@ -5,7 +5,13 @@
 import time
 from uiplib.scrape import get_images
 from threading import Thread
-
+import sys
+from select import select
+try:
+    import msvcrt
+except ImportError:
+    #not on windows
```
Comment (abhsag24) [inline uiplib/scheduler.py]: are u sure this is not on windows? :( how do we support windows then?

Comment (DarkSouL11) [inline uiplib/scheduler.py]: No msvcrt module is available only on windows :(
But I have defined functions for mac and linux

Comment (abhsag24) [inline uiplib/scheduler.py]: ohh nice

Comment (abhsag24) [inline uiplib/scheduler.py]: is it a standard library or do u have to install it?

Comment (DarkSouL11) [inline uiplib/scheduler.py]: yes it is standard library for windows.

Comment (abhsag24) [inline uiplib/scheduler.py]: yes that's fine :D

Comment (nkprince007) [inline uiplib/scheduler.py]: It works on macOS...

**Comment by pandusonu2** [comment]
:heart_eyes:

**Pull request merged**